app/routes/datamart/land.py

```diff
@@ -141,7 +225,7 @@ async def tree_cover_loss_by_driver_post(
     background_tasks.add_task(
```

Comment:
I think you might be missing one big part, which is actually modifying the task to deal with GADM areas per the method discussed here: https://gfw.atlassian.net/browse/GTC-3116

Basically, the bounds for an admin area might be too large, and we don't want to pass simplify boundaries to keep accuracy. So we need to pull simplified boundaries, buffered boundaries from the database, and then pass them to OTF with a filter using the rasterized GADM.

We can potentially merge this as is, and then add that optimization afterwards, although some countries with complicated bounds might not work (they'll hit a lambda limit).

Reply:
Or alternatively, pass a simplification parameter when getting the geostore just so it'll always work, but then we'll need to do some invalidation later once we have the full method since results might not be accurate.

Reply:
🤦 I remember us talking about this, @jterry64. 
Can we meet up after today's Drivers/Datamart huddle to revisit what an implementation might look like?
I remember suggesting a new implementation of `_compute_tree_cover_loss_by_driver`

Reply:
I'm interested in the solution to this as well. Mind if I join in?

Reply:
We decided to break this functionality into its own Jira ticket: https://gfw.atlassian.net/browse/GTC-3219